### ISSUE TITLE
Deathbound Ward

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5912]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The issue with this npc is that when he's cced with shackle undead from priest for example he will still use his cleave ability that will deal damage on a random player even tho he's imobilized and cant do anything else
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. shackle undead the npc
2. see how it still does cleave animation and damage
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
npc ID 37007
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/
https://www.wowhead.com/npc=37007/deathbound-ward
**Include Screenshots from the issue if necessary**
